unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AudioSource/Tests/ECSAudioSourceShould.cs

```diff
@@ -37,7 +33,14 @@ protected override IEnumerator SetUp()
             gameObject = new GameObject();
             entity = Substitute.For<IDCLEntity>();
             scene = Substitute.For<IParcelScene>();
-            audioSourceComponentHandler = new ECSAudioSourceComponentHandler(DataStore.i,Settings.i, AssetPromiseKeeper_AudioClip.i, CommonScriptableObjects.sceneNumber, Substitute.For<IInternalECSComponent<InternalAudioSource>>());
+            Settings.CreateSharedInstance(new DefaultSettingsFactory());
+            audioSourceComponentHandler = new ECSAudioSourceComponentHandler(
+                DataStore.i,
+                Settings.i,
```

Comment:
Good point, let me try

Reply:
Damn, that won't work here because the handler uses several audio values from the settings and I need to feed it the Settings instance instead of the factory so I added that `Settings.CreateSharedInstance()` to have deterministic player settings when running those tests (same as `ECSAudioSourceShould` and `VoiceChatWindowControllerShould` do).